### ISSUE TITLE
20180828 - Comment/Questions from Plumbing Review

### ISSUE THREAD

**Issue by theoryshaw** [OpeningDesign/LaundryMat#2]
Hi @rvzenteno,

- MAHs and MATs washers have 3/4" supplies, and 3/4" drains.
  - for the existing bulkheads in storage (see videos), the 3/4" hoses are grouped and drained into 4 1/2" standpipe drains.  Is this still allowable by code? @apbock?
- MFRs and T-400s have 3/4" supplies, and 3" drains.
  - Because these units have larger 3" drains, I'm not sure a standpipe approach would work.  @apbock?
- No need to design plumbing for existing restroom, or floor drains in existing utility room see: http://sketch.openingdesign.com/d/JWwniWsv1y
- In the latest commit https://github.com/OpeningDesign/LaundryMat/commit/35b9a36b63e17a3b65fc46fccdd0baed4490fe3e I've updated some of the washer revit families.  They had incorrect connector sizes.  I also ran an OCR on the PDF manuals, so they're searchable now.

**Comment by rvzenteno** [comment]
Hi @theoryshaw,

1. Do you have a picture of this? If it is the video that is shows the discharge to a trench, where is this trench? is it at every collection of Washers?
2. For the MFRs & T-400s was the plan to use the assemblies you show on this sketch: http://sketch.openingdesign.com/d/ss4FSehBNq
3. No problem I'm taking out the plumbing there.
4. I'll look through the things on the commit 35b9a36

**Comment by theoryshaw** [comment]
@rvzenteno, did you end up zooming (w/mouse roller) out on this link? http://sketch.openingdesign.com/d/ss4FSehBNq  It will show you more drawings.

> For the MFRs & T-400s was the plan to use the assemblies you show on this sketch: 

Not sure I follow that question.  Can you rephrase?

---

Let me know if that link doesn't answer all your questions.

**Comment by rvzenteno** [comment]
For the Standpipe assembly, are the sizes shown on the screenshot correct? or is the large pipe 4" and the vents 2"

![image](https://user-images.githubusercontent.com/600617/45189559-35ab5d80-b1ff-11e8-8d5d-627215e5de23.png)

**Comment by theoryshaw** [comment]
from: LaundryMat\Research and CA\Research and Submittals\_Existing Conditions & Code Research\Existing Conditions\Existing Dimensions\Storage\Storage - Equipment Dimensions.pdf

![image](https://user-images.githubusercontent.com/507113/45193643-ae68e480-b214-11e8-9ff4-6aa1b7079a5b.png)

I'll go and verify on site tomorrow.  Do you need any other dimensions, while i'm there?

**Comment by theoryshaw** [comment]
Took some field measurements here: http://sketch.openingdesign.com/d/pq2WUMP2zy

Let me know if you need anything else.

**Comment by theoryshaw** [comment]
Zoom out, to see all measurements. :)
